apps/demos/Demos/Chat/Customization/Angular/app/app.component.html

```diff
@@ -0,0 +1,79 @@
+<div class="demo-container">
+  <div class="chat-container">
+    <dx-chat
+      #chat
+      [user]="currentUser"
+      [items]="messages$ | async"
+      [showAvatar] = "true"
```

Comment:
no need for spaces here. Please make code style consistent

Reply:
[a40d776](https://github.com/DevExpress/DevExtreme/pull/28482/commits/a40d77698d64bcb42c0340010e88cdccd0628c50)